projects/ziti-console-lib/src/lib/shared-assets/styles/global.scss

```diff
@@ -709,3 +724,25 @@ lib-form-field-container {
   }
 }
 
+.icon-close {
+  font-size: 25px;
```

Comment:
we'll get constants some day i hope... at least a global.scss is a step in the right direction!? :)